unlock-app/src/services/storageService.ts

```diff
@@ -3,6 +3,7 @@ import axios from 'axios'
 import { EventEmitter } from 'events'
 import { decodeToken } from 'react-jwt'
 import { generateNonce } from 'siwe'
+import { ToastHelper } from '../components/helpers/toast.helper'
```

Comment:
This is not the right approach. Let's move the notifications in the UI layer (Components) and keep this as the API wrapper only...